video/out/opengl/context_drm_egl.c

```diff
@@ -588,8 +593,9 @@ static void drm_egl_get_vsync(struct ra_ctx *ctx, struct vo_vsync_info *info)
 
 static bool drm_egl_init(struct ra_ctx *ctx)
 {
-    if (!vo_drm_init(ctx->vo))
+    if (!vo_drm_init(ctx->vo)) {
         goto err;
+    }
```

Comment:
Don't need to modify this.

Reply:
I was already here and it caused an inconvenience when I was debugging.

Reply:
Well sure I presume you added the brackets while debugging. But you don't need to actually add them to the code.